contrib/test_jenkins.sh

```diff
@@ -832,9 +833,9 @@ test_jucx() {
 	echo "1..2" > jucx_tests.tap
 	if module_load dev/jdk && module_load dev/mvn
 	then
-		pushd ../bindings/java/
+		pushd bindings/java/src/main/native
 		export UCX_ERROR_SIGNALS=""
-		JUCX_INST=$ucx_inst mvn clean test
+		JUCX_INST=$ucx_inst make test
```

Comment:
minor: can do `make -C bindings/java/src/main/native test`
also, why need to export JUCX_INST variable?

Reply:
`JUCX_INST` is path where installed libraries. After maven uses it to load ucx libraries to classpath when running tests: https://github.com/openucx/ucx/blob/master/bindings/java/pom.xml#L35

Maven understands env variables in upper case only. Plus to not confuse UCX, added JUCX, so no warning will show.

Reply:
who is setting JUCX_INST when not running jenkins?
Maybe we should use m4/autoconf to substitute pom.xml.in->pom_xml with `@libdir@`?

Reply:
When not in Jenkins it'll copy libs from their src folders:
https://github.com/openucx/ucx/blob/master/bindings/java/pom.xml#L31-L34

Any case `JUCX_INST` is not related to this PR - we can discuss and make it in separate PR if needed.

Reply:
right, it's not related to this PR. 
the separate PR should fix+test running UCX from installed location, without additional env vars.